### ISSUE TITLE
Quality of adversaries and authenticity of results

### ISSUE THREAD

**Issue by SachJbp** [jind11/TextFooler#24]
There seems to be a issue in a few adversaries.

For example: A claimed adversary from mr_bert.txt is:
orig sent (0):	to portray modern women the way director davis has done is just **unthinkable**
adv sent (1):	to portray modern women the way director davis has done is just **imaginable**

**unthinkable** and **imaginable** are antonyms which erroneously have high cosine similarity suggesting that those are synonyms. I suggest such examples should not be considered while evaluating the success rate of attack, as the human evaluation would clearly label it as positive (1) and not negative.

**Comment by jind11** [comment]
Yes, the human evaluation on polarity is not 100% due to these errors.

**Comment by SachJbp** [comment]
The ~13% after-attack accuracy reported considers such examples as success , which actually is not. I guess Human evaluation filter should finally govern the after-attack accuracy. Thanks.

**Comment by jind11** [comment]
Human evaluation can check whether these "successful" examples are legitimate or not.




            
            
                
                    
                            
                                
                            
                            
                                Di Jin
                                    PhD student
                            
                    
                        
                            
                                    MIT
                                    邮箱：jindi15@mit.edu
                            
                        
                
            
        Signature is customized by Netease Mail Master


On 06/20/2020 16:09, Sachin Saxena wrote: 
The ~13% after-attack accuracy reported considers such examples as success , which actually is not. I guess Human evaluation filter should finally govern the after-attack accuracy. Thanks.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/jind11/TextFooler/issues/24#issuecomment-647040332",
"url": "https://github.com/jind11/TextFooler/issues/24#issuecomment-647040332",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
